molgenis/molgenis-emx2
# fix: add search message for biobanks

What are the main changes you did:
- closes  #3597 

how to test:
- select a filter e.g. biobank services 
- see message on the biobank card
- If a filter is on the biobank and a colllection the message is shown on both
- If a filter is on either it is shown on the one matching

todo:
